addons/mail/static/tests/mock_server/mock_models/mail_scheduled_message.js

```diff
@@ -22,7 +22,10 @@ export class MailScheduledMessage extends models.ServerModel {
                 attachment_ids: mailDataHelpers.Store.many(
                     IrAttachment.browse(message.attachment_ids)
                 ),
-                author: mailDataHelpers.Store.one(ResPartner.browse(message.author_id)),
+                author_id: mailDataHelpers.Store.one(
+                    ResPartner.browse(message.author_id),
+                    makeKwArgs({ fields: ["name", "email"] })
```

Comment:
email was for the notification PR no? schedule has something different in python